packages/app-project/src/screens/ProjectHomePage/components/ZooniverseTalk/components/RecentSubjects/components/Placeholder/Placeholder.js

```diff
@@ -0,0 +1,47 @@
+import { Box, Grid } from 'grommet'
+import React from 'react'
+import { WaveLoading } from 'styled-spinkit'
+import { withTheme } from 'styled-components'
+
+function Placeholder (props) {
+  const { theme } = props
+  const spinnerColour = theme.dark
+    ? theme.global.colors['light-5']
+    : theme.global.colors['light-4']
+
+  return (
+    <Grid
+      columns={['1fr', '1fr', '1fr']}
+      fill
+      gap='small'
+      {...props}
+    >
+      {[null, null, null].map((element, index) => (
```

Comment:
I didn't catch this on review to add the carousel, but perhaps on small sizes, this should only show a single loading placeholder, so it has the correct single column layout. Right now on mobile sizes this is what this looks like (took the screenshot while throttling):

<img width="347" alt="Screen Shot 2019-08-30 at 12 12 52 PM" src="https://user-images.githubusercontent.com/5016731/64039129-907c4a00-cb1f-11e9-8a27-a0cbd2340f1c.png">

Reply:
Yeah, the placeholder should be moved down the tree. Doesn't the Media component already have a placeholder?

Reply:
I've removed the placeholder.